well-known-extension-types.md

```diff
@@ -0,0 +1,5 @@
+This is a list of well known extension types. 
+
+| Type | Description | Further information |
+|:--- |:--- |:--- |
+|TREX | T-REX (Trivial Record Exchange) is a simple text-based serialization format. It is purpose-built for replacing manual transcription scenarios with easy to implement ad-hoc data transmissions.| https://github.com/ApiniLabs/T-REX|
```

Comment:
Might be worthwhile mentioning that T-REX is usually used for providing a generic summary of what the PAC-ID is pointing at.

Reply:
In addition, to save characters, I recommend that we use "T" or "TX" instead of "TREX" as type.